apps/schema/src/components/SchemaView.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div>
+    <div id="molgenis_tables_container">
+      <span v-if="schema.tables && schema.tables?.length > 0">
+        <span
+          v-for="(table, index) in schema.tables"
+          :key="JSON.stringify(table) + schema.tables.length + '_' + index"
```

Comment:
this might now work properly I will test. But previously, the view didn't update if one of the properties of the table was changed (e.g. a column or a description) unless I added the state to the key.

Reply:
indeed, is legacy from previous schema editor. fixed now.